cc-mt-service/README.md

```diff
@@ -7,3 +7,8 @@
    - ```node dist/app.js```. Runs the created JavaScript file with Node
 
 Additional script: Run ```npm run check``` to check the file for TypeScript errors/warnings with eslint
+
+### Docker Guide 
```

Comment:
Maybe add note that you should add .env file before

Reply:
done